ui/src/api/index.js

```diff
@@ -0,0 +1,7 @@
+import wretch from 'wretch';
+
+const api = wretch('/mxcube/api/v0.1')
+  .options({ credendials: 'include' })
+  .headers({ Accept: 'application/json' });
```

Comment:
Here, I prepare a `wretch` instance with the base URL of the API server and with some default options/headers. This is where we might register middleware in the future.